src/services/createCampaignForGivethioProjects/createCampaignForGivethioProjects.service.js

```diff
@@ -0,0 +1,99 @@
+const errors = require('@feathersjs/errors');
+const config = require('config');
+const logger = require('winston');
+const { findCampaignByGivethIoProjectId } = require('../../repositories/campaignRepository');
+const { getGivethIoAdapter } = require('../../adapters/adapterFactory');
+
+const givethIoAdapter = getGivethIoAdapter();
+module.exports = function aggregateDonations() {
+  const app = this;
+
+  const service = {
+    async create(data, params) {
+      const { txHash, image, slug } = data;
+
+      const projectInfo = await givethIoAdapter.getProjectInfoBySLug(slug);
+      const {
+        id: givethIoProjectId,
+        title,
+        description,
+        walletAddress: ownerAddress,
+      } = projectInfo;
+      if (params.user.address !== ownerAddress) {
```

Comment:
We don't know ownerAddress in giveth.io backend has correct checksum
```suggestion
      if (params.user.address.toLowerCase() !== ownerAddress.toLowerCase()) {
```

Reply:
Good point,
done